conference/tests/test_views.py

```diff
@@ -0,0 +1,246 @@
+        conference = ConferenceFactory()
+        schedule = ScheduleFactory(conference=conference.code)
+
+        url = reverse('conference-schedule-xml', kwargs={
+            'conference': conference.code,
+            'slug': schedule.slug,
+        })
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.get('content-type'), 'application/xml')
+        self.assertEqual(response.context['schedule'], schedule)
+
+    @unittest.skip('todo')
+    def test_conference_schedule(self):
+        # conference-schedule -> conference.views.schedule
+        url = reverse('conference-schedule')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_schedule_event_booking(self):
+        # conference-schedule-event-booking -> conference.views.schedule_event_booking
+        url = reverse('conference-schedule-event-booking')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_schedule_event_interest(self):
+        # conference-schedule-event-interest -> conference.views.schedule_event_interest
+        url = reverse('conference-schedule-event-interest')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_schedule_events_booking_status(self):
+        # conference-schedule-events-booking-status -> conference.views.schedule_events_booking_status
+        url = reverse('conference-schedule-events-booking-status')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_schedule_events_expected_attendance(self):
+        # conference-schedule-events-expected-attendance -> conference.views.schedule_events_expected_attendance
+        url = reverse('conference-schedule-events-expected-attendance')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_speaker(self):
+        # conference-speaker -> conference.views.speaker
+        url = reverse('conference-speaker')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    @unittest.skip('todo')
+    def test_conference_speaker_xml(self):
+        # conference-speaker-xml -> conference.views.speaker_xml
+        speaker = SpeakerFactory()
+
+        url = reverse('conference-speaker-xml', kwargs={
+            'slug': '123',
+        })
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.get('content-type'), 'application/xml')
+        self.assertEqual(response.context['speaker'], speaker)
+
+    @override_settings(DEBUG=False)
```

Comment:
this sound strange, why are we overriding debug to be false? 
Ideally should always be false :)